services/resourcemanager/wait/wait_test.go

```diff
@@ -91,7 +91,7 @@ func TestCreateProjectWaitHandler(t *testing.T) {
 
 			handler := CreateProjectWaitHandler(context.Background(), apiClient, "cid")
 
-			gotRes, err := handler.SetTimeout(10 * time.Millisecond).WaitWithContext(context.Background())
+			gotRes, err := handler.SetTimeout(10 * time.Millisecond).SetSleepBeforeWait(10 * time.Millisecond).WaitWithContext(context.Background())
```

Comment:
The default `SetSleepBeforeWait` for this wait handler is 1 minute, meaning this test alone (with it's 4 test cases) was taking more than 4 minutes